package.json

```diff
@@ -34,7 +34,8 @@
     "mkdirp": "^0.5.1",
     "ncp": "^2.0.0",
     "node-notifier": "^5.3.0",
-    "tslib": "^1"
+    "tslib": "^1",
+    "yarn": "^1.15.2"
```

Comment:
You have added `yarn` as a dependency of the project here. I suspect that in the process of updating `yarn` on your laptop you have added it as a chectl dependency as well (i.e. you have run `yarn add yarn`). If that's the case you should revert it (using command `yarn remove yarn`) and update your commit.

Reply:
I didn't add it explicitly but given the build DOES require yarn why is it bad to state it here?

Reply:
Do you usually add maven in your pom.xml?

Reply:
no, just maven plugins... so if here you're saying it's not valuable to define a minimum version of yarn that's been tested to work here, I'll remove it.

Reply:
@nickboldt please remove `yarn` as a dependency so that we can proceed merging this PR

Reply:
I think I already did last week, but missed a reference in yarn.lock. Force pushed an update to this PR. Is that better?